example/workflows.yml

```diff
@@ -545,10 +545,11 @@ KinesisTriggerTest:
         cumulus_message:
           input: '{$}'
       Catch:
-        - ErrorEquals:
+        - &ErrorEqualDefaults
```

Comment:
Is the `&` or `<<: *ErrorEqualDefaults` syntax explained/defined anywhere? I haven't seen this before.

Reply:
Yup: https://blog.daemonl.com/2016/02/yaml.html - I think we should use these shorthands more since these workflows are pretty repetitive. I added a note about them to the data cookbook as well since others will probably have the same question: https://github.com/nasa/cumulus/pull/494/commits/ba3496dcf4e0a840d91ad131f7c12ceb6ff29aec